src/redux/reducers/loginSlice.jsx

```diff
@@ -0,0 +1,55 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { instance } from "../../utilities";
+
+const initialState = {
+  user: null,
+  isVerified: false,
+};
+
+export const signup = async (username, password) => {
+  try {
+    const response = await instance.post(
+      `add_user?username=admin&password=admin&role=admin`,
+      {
+        username,
+        password,
+      }
```

Comment:
why need to pass body
also string literals are wrong
we pass like `password=${admin}`